sdk/identity/test-resources-pre.ps1

```diff
@@ -5,6 +5,13 @@ param (
     $RemainingArguments
 )
 
+Import-Module -Name $PSScriptRoot/../../eng/common/scripts/X509Certificate2 -Verbose
```

Comment:
@benbp is there a better way to do this with an $engsyscommon define so scripts don't have to ../../ back? this doesnt seem ideal.

Reply:
@joshfree We could do `$(System.DefaultWorkingDirectory)/eng/common/...` that way it's an absolute path, but it wouldn't work locally. Similarly, any defines would need to be imported in the powershell script anyway, which would require a similar type of reference. I agree it's not ideal to have the relative pathing, but short of pipeline only variables or publishing modules, I'm not sure there's a great solution.

Reply:
Thanks for checking it out